front/app/modules/commercial/insights/admin/components/Tag/index.tsx

```diff
@@ -35,25 +39,46 @@ const CloseIcon = styled(Icon)`
   fill: #fff;
 `;
 
-const StyledTag = styled.div<{ status: Status }>`
-  ${({ status, theme }) => css`
+const Count = styled.div`
+  margin-left: 8px;
+`;
+
+const StyledTag = styled.button<{ variant: Variant }>`
+  ${({ variant, onClick, theme }) => css`
     border-radius: ${theme.borderRadius};
     cursor: default;
     font-size: ${fontSizes.small}px;
     font-weight: normal;
     display: inline-block;
     padding: 4px 12px;
-    ${status === 'approved' &&
-    css`
-      background-color: ${colors.clGreen};
-      color: #fff;
-    `}
-    ${status === 'suggested' &&
-    css`
-      background-color: #fff;
-      color: ${colors.label};
-      border: 1px solid ${colors.border};
-    `}
+    ${
+      variant === 'primary' &&
+      css`
+        background-color: ${colors.clGreen};
+        border: 1px solid ${colors.clGreen};
+        color: #fff;
+      `
+    }
+    ${
+      variant === 'secondary' &&
+      css`
+        background-color: #fff;
+        color: ${colors.label};
+        border: 1px solid ${colors.border};
+      `
+    }
+    ${
+      onClick &&
+      css`
+        cursor: pointer;
+        &:hover {
```

Comment:
I always like when the focus style is similar to hover style, well except for the outline, but I'm not sure how it's done everywhere else... Anyways, my personal preference would be to also darken on focus :  
```suggestion
        &:hover, &:focus {
```
Do with that what you deem best :)

Reply:
Yes, agreed!